src/plugins/ScryptEncrypter.ts

```diff
@@ -68,46 +60,42 @@ export const ScryptEncrypter = {
   async encryptKey({
     key,
     password,
+    salt,
+    nonce,
   }: {
     key: Key;
     password: string;
+    salt?: string;
```

Comment:
I'd make it clear that salt and nonce should not be set for anything but testing, since it's a pretty bad mistake to set them in production.

Reply:
I moved the scrypt logic into its own lib, the lib takes the overrides now.